### ISSUE TITLE
Create record in database

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#23]
## User story
1. As a dev
2. I want to 
3. So that {}

## Acceptance criteria
* A record saved in DB of the PDF (location of the PDF)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
